replicator-security/README.md

```diff
@@ -0,0 +1,128 @@
+# Replicator security configuration demos
+
+## Purpose 
+
+This repository provides docker-compose environments representing various Replicator security configurations. 
+
+## Non replicator components
+
+All environments contain:
+
+* A 3 node source cluster containing srcKafka1, srcKafka2 and srcKafka3
+* A 3 node destination cluster containing destKafka1, destKafka2, destKafka3
+* A single Connect worker that will run replicator
+* A source client container (srcKafkaClient) that creates a source topic and provides test data
+* A destination client container (destKafkaClient) that installs Replicator
+
+In all containers, minimal non-replicator configuration is supplied.
+
```

Comment:
Explain more about Replicator -- just a few lines:

* What is Confluent Replicator
* Where it runs
* Why it needs to know source security settings
* Why it needs to know destination security settings
* Provide link to https://docs.confluent.io/current/installation/docker/installation/replicator.html for more in-depth understanding of how Replicator works in mdc env

Reply:
updated, please let me kniow if i can add anything more